.github/workflows/ci-docs.yml

```diff
@@ -0,0 +1,38 @@
+# This workflow will install Python dependencies, and build the website
+
+name: CI - Documentation Website
+
+on:
+  # Trigger the workflow on push or pull request,
+  # but only for the master branch
+  push:
+    branches:
+      - main
+  pull_request:
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: ['3.13']
+
+    steps:
+    - uses: actions/checkout@v4
+
+    - name: Install uv
+      uses: astral-sh/setup-uv@v5
+      with:
+        enable-cache: true
+        cache-dependency-glob: "uv.lock"
+
+    - name: Set up Python ${{ matrix.python-version }}
+      run: uv python install ${{ matrix.python-version }}
+
+    - name: Install the project
+      run: VENV_EXTRAS="--all-extras" make venv
+
+    - name: Build documentation
+      run: uv run mkdocs build --strict
```

Comment:
is the `--strict` needed here? I thought having it in the [yaml](https://www.mkdocs.org/user-guide/configuration/#strict) file was enough?

Reply:
I'm not 100% sure, seems fine to keep it in both places to me, but it's a good point